### PR TITLE
add notes and questions, n25325

### PR DESCRIPTION
**Pull request by LarryRuane**

### CODE REVIEW

**Review by stickies-v — APPROVED** [review]
Thanks @LarryRuane ! Did a rather quick review while traveling, so couldn't look at the underlying PR. Added some small fixes (we usually use unnumbered list for notes) but overall I think this looks good!

Main comment would be that the notes don't mention pool based memory resources at all. Given that it's the topic of the PR, perhaps that would be useful to provide some more context on? But I think this is good to merge already so for the sake of time I'll do that already and feel free to follow with another pull if you think it makes sense?

**Comment by LarryRuane** [comment]
@stickies-v @jonatack 
Thanks, I just opened #635 to implement your suggestions

File: _posts/2023-03-08-#25325.md [hunk 1]
```diff
@@ -7,19 +7,111 @@ authors: [martinus]
...
+   the codebase as a
+   [`Coin`](https://github.com/bitcoin/bitcoin/blob/3b88c8502534f0dc94e0abcb04ffa80ba8bd7f01/src/coins.h#L30) object.
+   The key information in a `Coin` is a
+   [`CTxOut`](https://github.com/bitcoin/bitcoin/blob/3b88c8502534f0dc94e0abcb04ffa80ba8bd7f01/src/primitives/transaction.h#L157),
+   which contains an amount (`nValue`) and a `scriptPubKey`, sometimes referred
+   to as the _locking script_ because it prevents the coin from being
+   spent without the authorizing witness as an input in a later transaction.
+
+- For inclusion in the coins cache, each `Coin` is wrapped within
+   a `CCoinsCacheEntry`(https://github.com/bitcoin/bitcoin/blob/8126551d54ffd290ed5767248be4b3d19243787b/src/coins.h#L103)
```
Comment (glozow) [inline _posts/2023-03-08-#25325.md:64]: Need square brackets to embed the link
```suggestion
   a [`CCoinsCacheEntry`](https://github.com/bitcoin/bitcoin/blob/8126551d54ffd290ed5767248be4b3d19243787b/src/coins.h#L103)
```

Comment (LarryRuane) [reply _posts/2023-03-08-#25325.md:64]: Thanks, just created #636

**Pull request merged**